db/migrate/20240429133246_remove_start_date_and_end_date_from_placements.rb

```diff
@@ -0,0 +1,7 @@
+class RemoveStartDateAndEndDateFromPlacements < ActiveRecord::Migration[7.1]
+  def change
+    change_table :placements, bulk: true do |t|
+      t.remove :start_date, :end_date, type: :date
+    end
```

Comment:
I think you'll need to update the `analytics.yml` files and remove these columns from there.

Reply:
Oh, you did preemptively in the commit before this change.